config/config.toml.example

```diff
@@ -109,7 +109,7 @@ repair-table-list = []
 max-server-connections = 0
 
 # Whether new collations are enabled, as indicated by its name, this configuration entry take effect ONLY when a TiDB cluster bootstraps for the first time.
-new_collations_enabled_on_first_bootstrap = false
+new_collations_enabled_on_first_bootstrap = true
```

Comment:
the naming of this configuration is different than any other configurations, is it more appropriate to rename it to

new-collations-enabled-on-first-bootstreap

Reply:
It's a sad story, we want to fix it, but not in this PR.